edith-step/src/UserChart.js

```diff
@@ -163,8 +164,22 @@ const PieChart = () => {
         console.log(response);
         let itemNames = [];
         let itemValues = [];  
-        const items = JSON.parse(response.data.items);
-        console.log(items);
+        let items = [];
+        const itemsList = JSON.parse(response.data.items);
+        if (props.dateSelection != "") {
+          console.log(props.dateSelection);
+          for (let i = 0; i < itemsList.length; i++) {
+            inSameWeek(itemsList[i].date, props.dateSelection);
+            console.log(itemsList[i]);
```

Comment:
Same comment here and for the log statement on line 176

Reply:
I removed it.